koku/providers/provider_errors.py

```diff
@@ -78,7 +78,7 @@ class ProviderErrors:
         "Edit your Azure source to include these details."
     )
     AZURE_INCORRECT_CLIENT_SECRET_MESSAGE = (
-        "The client secret was entered incorrectly for this source. "
+        "The client secret was entered incorrectly or has expired for this source. "
```

Comment:
@dccurtis @nlcwong Maybe something like this:
A problem has been detected with the Azure client secret for this source. Refer to the Microsoft Azure troubleshooting guide in the cost management documentation for details.   
??? too long?

Reply:
I think that would be ok.  I can update the code and give an example in the UI next.  If we reference the docs we should wait until the docs are updated before making the error message update.

Reply:
@nlcwong I updated the screenshots with @xJustin wording suggestion.  How's it look to you?